### DISCUSSION TITLE
Input does not show value

### DISCUSSION THREAD

**Discussion by rodrigocabraln** [livewire/livewire#6998] [1. Help]
Trying to make an editing modal I can't get the data to load in the inputs, Livewire 3, Laravel 10.

simple example code.

Component:
```
<?php

namespace App\Livewire\Condominios;
use Livewire\Component;
use Livewire\Attributes\Rule;

class EditCondominio extends Component {

    #[Rule('required')]
    public $nombre2;


    public function render() {
        return view('livewire.condominios.edit-condominio-modal');
    }

    public function selectForEdit() {
        $this->nombre2 = "Pepe";
    }
}
```

View:
```
<div>

    <button type="button" class="bg-primary-700"
        wire:click="selectForEdit()">Editar
    </button>

    <input type="text" wire:model="nombre2" id="txtNombre2">

    <br>

    <label class="text-white">{{ $nombre2 }}</label>
</div>
```

Result:
![Captura desde 2023-10-04 13-27-46](https://github.com/livewire/livewire/assets/63825193/da4e4498-0d05-4e29-b0eb-8dd64abc696f)

If I do this it works, but I lose the data binding:

` <input type="text" value="{{$nombre2}}" id="txtNombre2">`

**Comment by mcpuishor** [comment]
I feel like the error must be somewhere else. The above code works perfectly. 
Have you checked if there are any errors reported in the DevTools console?

**Reply by rodrigocabraln** [reply]
Without errors in the console, If I use "model.live" what I write in the input is reflected in the label, but after clicking the button, the label changes with "Pepe", the input maintains the previous value.

I attach the response from the request after clicking the button

```
{
    "components": [
        {
            "snapshot": "{\"data\":{\"nombre2\":\"Pepe\"},\"memo\":{\"id\":\"jvpLTkygi8RO1NH2mG5q\",\"name\":\"condominios.edit-condominio\",\"path\":\"administracion\\\/condominios\",\"method\":\"GET\",\"children\":[],\"errors\":[],\"locale\":\"es\"},\"checksum\":\"459848a4e63c047616e83d83531d1870035ea0fad00f62f635a0acea36f31698\"}",
            "effects": {
                "returns": [
                    null
                ],
                "html": "<div wire:id=\"jvpLTkygi8RO1NH2mG5q\">\n\n    <button type=\"button\" class=\"bg-primary-700\"\n        wire:click=\"selectForEdit()\">Editar\n    <\/button>\n\n    <input type=\"text\" wire:model.live=\"nombre2\" name=\"nombre2\" id=\"txtNombre2\">\n\n    <br>\n\n    <label class=\"text-white\">Pepe<\/label>\n<\/div>\n"
            }
        }
    ]
}
```

**Reply by RoccoHoward** [reply]
Got the exact same issue - just upgraded from v2 and v3 and have the same thing happening. Not sure what's going on - the input holds onto the value, but livewire clears the property value any "use", so is definitely related to something with how the input update occurs.

**Reply by rodrigocabraln** [reply]
I abandoned LiveWire, I can't find a solution

**Reply by joshhanley** [reply]
I just tested the component supplied at the top and it works happily for me. @RoccoHoward and @rodrigocabraln can you make sure you are on the latest version of Livewire v3.0.10 and make sure you aren't including Alpine anywhere in your layout or `app.js` file?

If one of you can provide a repo demonstrating the issue, I'd be happy to have a look at it for you.

**Reply by RoccoHoward** [reply]
Hi @joshhanley That's the issue. Confirmed now working after removing Alpine in addition to including Livewire v3. 

Probably a few of us going get caught once of that one.

Sorry for wasting your time and appreciate your follow-up.

**Reply by injektion** [reply]
For those who've ended up here, Livewire 3 sets up Alpine by default.

If you've used Breeze/Inertia there's a chance it's installed Alpine for you already.

Go into App.js, and if you see this:


```
import './bootstrap';

import Alpine from 'alpinejs';

window.Alpine = Alpine;

Alpine.start();
```


... remove all the Alpine stuff from there and it'll work


`import './bootstrap';`


(Spent 4 hours trying to figure out why my input field wouldn't update when I made changes to the variable for it to be just that...)

**Reply by craigfanatic** [reply]
So much love for you here. This worked a charm for me struggling with nested data binding between a parent and child Livewire component.
